src/main/java/uk/gov/hmcts/ccd/data/casedetails/CaseAuditEventEntity.java

```diff
@@ -34,6 +34,17 @@
     ),
     @NamedQuery(name = CaseAuditEventEntity.FIND_BY_ID, query =
         FIND_BY_ID_HQL
+    ),
+    @NamedQuery(name = CaseAuditEventEntity.FIND_BY_CASE_REFERENCE, query =
+        "SELECT cae FROM CaseAuditEventEntity cae "
```

Comment:
what about the security checks around security classification that are carried out by the `ClassifiedGetCaseOpeartion`? Those are not in the query so are we losing them?
See:
https://github.com/hmcts/ccd-data-store-api/blob/87876fffbbc3f356f62a581c57604d39de89ef3c/src/main/java/uk/gov/hmcts/ccd/domain/service/common/SecurityClassificationService.java#L57